cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -642,6 +642,7 @@ func EnsureAdminClusterRoleBindingImpl(ctx context.Context, adminClient, superAd
 				clusterRoleBinding,
 				metav1.CreateOptions{},
 			); err != nil {
+				crbResult = nil
```

Comment:
this almost feels like a bug or a "miss-design" in client-go.
in go stdlib commonly if err != nil, everything else is nil or 0 value.

Reply:
Yeah; I would have changed the real clients to return nil rather than changing the fake clients to return non-nil but that would presumably break real code...